src/Psalm/Internal/Analyzer/ClassAnalyzer.php

```diff
@@ -211,6 +211,20 @@ public function analyze(
                     }
                 }
 
+                if ($parent_class_storage->psalmInternal &&
+                    strpos($fq_class_name, trim($parent_class_storage->psalmInternal, '\\') . '\\') !== 0
+                ) {
+                    if (IssueBuffer::accepts(
```

Comment:
I followed the practice of wrapping calls to `IssueBuffer::accepts` in `if`, but I'm wasn't clear on the purpose of the `if`.